app/assets/stylesheets/searchworks4/homepage.css

```diff
@@ -75,6 +75,10 @@
     font-weight: 700;
     color: var(--stanford-cardinal);
   }
+  .tips span {
+    color: var(--bs-header-color);
```

Comment:
Where is `--bs-header-color` defined?  I don't see it being set anywhere.

Reply:
I can just set it to black rather than the variable. Without setting the color it defaults to `--bs-body-color` which to me looks lighter than the provided design. If you have a particular way you would like me to set black, let me know.